lte/gateway/c/oai/tasks/sgw/pgw_handlers.h

```diff
@@ -46,6 +46,6 @@ int32_t spgw_handle_nw_initiated_bearer_deactv_req(
     imsi64_t imsi64);
 
 int spgw_send_nw_init_activate_bearer_rsp(
-    gtpv2c_cause_value_t cause, imsi64_t imsi64, uint8_t eps_bearer_id,
-    uint8_t default_bearer_id, char* policy_rule_name);
+    gtpv2c_cause_value_t cause, imsi64_t imsi64, const char* imsi,
```

Comment:
Passing both imsi64 and imsi seems overloaded. Can we use IMSI64_TO_STRING macro to convert "imsi64" parameter to "char* imsi"?